src/main/java/ru/itmo/sd/deadliner2bot/model/Chat.java

```diff
@@ -0,0 +1,21 @@
+package ru.itmo.sd.deadliner2bot.model;
+
+import javax.persistence.*;
+import java.math.BigInteger;
+import java.util.Set;
+
+@Entity
+@Table(name = "chats")
+public class Chat {
+
+    @Id
+    @GeneratedValue
+    @Column(name = "chat_id")
+    private BigInteger chatId;
```

Comment:
Are you sure that PostgreSQL's `BIGINTEGER` type is an unlimited integer number?

Reply:
Passive agression review...

Reply:
I was not sure on my own at the moment of my last comment, sorry.
Please user `long` instead.